src/main/java/eu/deltasource/internship/hotelmanagementsystem/hotel/service/domain/commodities/Bed.java

```diff
@@ -0,0 +1,40 @@
+package eu.deltasource.internship.hotelmanagementsystem.hotel.service.domain.commodities;
+
+/**
+ * Class bed has 1 private member and one inherited from the abstract class AbstractCommodity
+ * methods - getters,setters,constructor and implemented method prepare()
+ */
+
+
+public class Bed extends AbstractCommodity {
+
+    private BedSize numberOfPersonas;
+
+
+    /**
+     * Parametrized constructor
+     *
+     * @param numberOfPersonas bed capacity
+     */
+
+    public Bed(BedSize numberOfPersonas) {
+        super();
+        this.numberOfPersonas = numberOfPersonas;
+    }
+
+    public int getNumberOfPersonas() {
+        return numberOfPersonas.getSize();
+    }
+
+    public void setNumberOfPersonas(BedSize numberOfPersonas) {
```

Comment:
We already have a way to set the type in the constructor, let's keep this class immutable and remove this setter, because there should not be a reason that a bed can become from a single to double, right?